src/value.rs

```diff
@@ -61,7 +61,7 @@ use super::unsafe_string::IString;
 ///   six JSON types.
 #[repr(transparent)]
 pub struct IValue {
-    ptr: NonNull<u8>,
+    pub(crate) ptr: NonNull<u8>,
```

Comment:
why does this need to be `pub(crate)`? Can't we use `ptr()` to get the pointer as `*mut u8` without the tag info?